.github/workflows/test.yml

```diff
@@ -144,7 +144,7 @@ jobs:
         run: echo "RUST=$(rustc --version)" >> "$GITHUB_ENV"
 
       - name: Upload to codecov.io
-        uses: codecov/codecov-action@v4
+        uses: codecov/codecov-action@v5
```

Comment:
A bug with [env_vars: OS,RUST](https://github.com/OpenZeppelin/rust-contracts-stylus/blob/main/.github/workflows/test.yml#L151) causing a [failed workflow](https://github.com/OpenZeppelin/rust-contracts-stylus/actions/runs/11849225069/job/33022114270).
The bug was [already reported](https://github.com/codecov/codecov-action/issues/1649), and the fix should be [released tomorrow](https://github.com/codecov/codecov-action/issues/1649#issuecomment-2477891010).

Reply:
@0xNeshi are you following this?

Reply:
Thanks for the reminder, on it!